roles/edpm_container_quadlet/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Copyright 2025 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# "edpm_container_quadlet" will search for and load any operating system variable file
+
+- name: "Ensure directory exists: {{ edpm_container_quadlet_kolla_config_dir }}"
+  become: true
+  ansible.builtin.file:
+    path: "{{ edpm_container_quadlet_kolla_config_dir }}"
+    state: directory
+    recurse: true
+    setype: container_file_t
+
+- name: Create kolla config files
+  become: true
+  ansible.builtin.copy:
+    content: "{{ item.value | to_nice_json }}"
+    dest: "{{ edpm_container_quadlet_kolla_config_dir ~ '/' ~ item.key ~ '.json' }}"
+    mode: "0600"
+  loop: "{{ edpm_container_quadlet_kolla_config_files | dict2items }}"
+
+- name: Copy systemd service template file
+  ansible.builtin.copy:
+    src: "files/service-template.kube"
+    dest: /etc/containers/systemd/edpm-compute@.kube
```

Comment:
I'd drop the `-compute` and just use `edpm@.kube`. just b/c there can be other edpm node types besides compute.